src/codewhisperer/models/constants.ts

```diff
@@ -153,6 +155,9 @@ export const defaultCheckPeriodMillis = 1000 * 60 * 5
 // suggestion show delay, in milliseconds
 export const suggestionShowDelay = 250
 
+// add 200ms more delay on top of inline default 30-50ms
+export const inlineSuggestionShowDelay = 200
```

Comment:
Why are the new constants needed? They are similar to existing constants. Over-granular configuration like this can be hard to work with.

Reply:
the value is slightly different from the existing one used in the old code path without VSC inline api. This is trying to match the actual delay time to the old path, as well as JB as close as possible.